scout/server/blueprints/alignviewers/controllers.py

```diff
@@ -113,7 +113,7 @@ def make_igv_tracks(case_obj, variant_id, chrom=None, start=None, stop=None):
     return display_obj
 
 
-def make_sashimi_tracks(case_obj, variant_id):
+def make_sashimi_tracks(case_obj, variant_id=None):
```

Comment:
Perhaps split this complex function into smaller ones, as also codefactor suggests? Or even have a distinct function for when there is no variant id? Both function can call the same smaller functions..